ui/app/components/clients/dashboard.js

```diff
@@ -3,7 +3,7 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { format } from 'date-fns';
 
-export default class HistoryComponent extends Component {
+export default class DashboardComponent extends Component {
```

Comment:
I noticed the generator labels components `Dashboard` - (without `Component`) not sure if we want to follow with that convention?